src/Core/Managed/Shared/Extensibility/Implementation/Tracing/CoreEventSource.cs

```diff
@@ -399,6 +399,26 @@ public void FailedToLoadEnvironmentVariables(string ex, string appDomainName = "
             this.WriteEvent(32, ex, this.nameProvider.Name);
         }
 
+        // Verbosity is Error - so it is always sent to portal; Keyword is Diagnostics so throttling is not applied.
+        [Event(33,
+            Message = "Metrics Extractor should be used before Sampling Processors or any other Telemetry Processors that might filter out Telemetry Items. Otherwise, extracted metrics may be incorrect.",
```

Comment:
can you state a problem first. "Metric Extractor detected an element with SamplingPercentage set to {0}. Metrics Extractor should be used before Sampli..."